modin/core/execution/ray/generic/io/io.py

```diff
@@ -196,3 +197,77 @@ def func(df, **kw):
         for rows in result:
             for partition in rows:
                 wait([partition.oid])
+
+    @staticmethod
+    def _to_parquet_check_support(kwargs):
+        """
+        Check if parallel version of ``to_parquet`` could be used.
```

Comment:
```suggestion
        Check if parallel version of `to_parquet` could be used.
```
We use one back stick for methods and two ones for classes.